src/executors/default.yml

```diff
@@ -3,6 +3,7 @@ description: >
 
 docker:
   - image: 'cimg/python:<<parameters.tag>>'
+resource_class: << parameters.resource_class >>
```

Comment:
This `resource_class` would get overridden by the one specified at a job level, so it doesn't do anything.

Reply:
@kynefuk could you update your PR?